packages/Checkbox/src/Checkbox.js

```diff
@@ -41,8 +31,11 @@ const Checkbox = props => {
     size,
   };
 
+  if (extendedInputProps.refLabel) {
+    delete extendedInputProps.refLabel;
+  }
+
   const inputProps = {
-    "aria-describedby": ariaDescribedBy,
```

Comment:
This isn't needed anymore? It's a breaking change – should be added to the `CHANGELOG`

Reply:
Yes, not necessary when we are spreading the `a11yProps` on the `Checkbox.Input` itself (https://github.com/acl-services/paprika/pull/774/files/d3d5d79b37a2579b3b51be592fca03140a7395d0#diff-412375913c3db82b1954e7b42b97f36557cef23c5dbb7c5bcae4fe3d00066177R16)